File: C2009i/AP/27-09-2021/SalaryApp/SalaryApp/Models/Employee.cs
using System;
namespace Aptech.Models
{
    public class Employee: Person
    {
        public double Salary { get; set; }
        public void IncreaseSalary(int x) {
            Salary += x;
        }
        
        public void IncreaseSalary(double x)
        {
            Salary *= x;
        }
        public override void Display()
        {
            base.Display();
            Console.WriteLine($"Salary = {Salary}");
        }
    }
}

File: C2108G1/ADF1/01-10-2022/studentapp/src/main/java/com/aptech/Employee.java
package com.aptech;

public class Employee extends Person{
    private int salary;
    public Employee(){}
    public Employee(String name, Integer age, int salary) {
        super(name, age);
        this.salary = salary;
    }
    //overload
    public void increaseSalary(int x){
        this.salary += x;
    }
    public void increaseSalary(double x){
        this.salary = (int)(this.salary * x);//explicit casting
    }

    public int getSalary() {
        return salary;
    }

    public void setSalary(int salary) {
        this.salary = salary;
    }

    @Override
    public void display() {
        super.display();
        System.out.println("salary:"+salary);
    }
}
